homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include "vectorFunctions.hpp"
+
+std::vector<std::shared_ptr<int>> generate(int count)
+{
+    if(count < 0)
+    {
+        count = 0;
+    }
+    
+    std::vector<std::shared_ptr<int>> vector;
+    vector.reserve(count);
+    for(size_t i {0}; i < count; ++i)
+    {
+        vector.push_back(std::make_shared<int>(i));
+    }
+    return vector;
+}
+
+void print(const std::vector<std::shared_ptr<int>>& t_Vector)
+{
+    for(const auto& pElement : t_Vector)
+    {
+        if(pElement != nullptr)
+        {
+            std::cout << *pElement << " ";
+        }
+    }
+}
+
+void add10(const std::vector<std::shared_ptr<int>>& t_Vector)
+{
+    for(const auto& pElement : t_Vector)
+    {
+        if(pElement != nullptr)
```

Comment:
if (value) is enough. This will call implicit `bool` operator which compare this ptr with `nullptr`

Reply:
Add space after `if`, `for`, `while` `function` etc...